README.md

```diff
@@ -69,6 +69,22 @@ Install Kerberos client to enable `skt` authentication to Beaker:
 Install `skt` dependencies following its
 [README.md](https://github.com/RH-FMK/skt).
 
+Run tests
+---------
+
+For running all tests write down:
+
+    python -m pytest -v -s tests
+
+For running some specific tests you can do this as following:
+
+    python -m pytest -v -s tests/test_patchwork
```

Comment:
This line complains about nonexistent file. Can you fix it so we have examples that work?

Reply:
I see it works in my environment. Are you sure you have `pytest` dependency?

Reply:
Yes I am. There is no `test_patchwork` file in the repository.

Reply:
`python -m pytest -v -s tests/test_patchwork.py`. Does it work with this command? I am not sure why it works with `test_patchwork` without `.py` in my environment